.github/workflows/ci-nestest.yaml

```diff
@@ -0,0 +1,60 @@
+name: Nestest CI
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+
+    env:
+      CTEST_OUTPUT_ON_FAILURE: ON
+      CTEST_PARALLEL_LEVEL: 2
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: nestest-clang-9
+            os: ubuntu-18.04
+            compiler: clang
+            version: "9"
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install
+        run: |
+          echo "::add-path::/home/runner/.local/bin"
+          python3 -m pip install pip --upgrade
+          python3 -m pip install cmake==3.14.4 ninja==1.9.0.post1 --upgrade
+
+          sudo apt update
+
+          sudo apt-get install -y clang-${{ matrix.version }} g++-multilib
+          echo "::set-env name=CC::clang-${{ matrix.version }}"
+          echo "::set-env name=CXX::clang++-${{ matrix.version }}"
+
+      - name: CMake
+        run: |
+          cmake --version
+          ninja --version
+          echo "CMake args: ${{ matrix.cmake-args }}"
+
+          cmake -S . -B build ${{ matrix.cmake-args }}
+
+      - name: Build
+        run: |
+          cmake --build build --config ${{ matrix.configuration }} --target nestest
+
+      - name: Run Nestest
+        run: |
+          wget http://nickmass.com/images/nestest.nes
+          wget http://www.qmtpro.com/~nes/misc/nestest.log
+          ./build/nestest/nestest nestest.nes | tee nestest-output.log
```

Comment:
Maybe run diff on the 2 logs too? That way we don't have to manually download and diff them like peasants. 😁

Reply:
That is what the python script it for :) Was planning on getting this merged first and then update once the script is merged. Unfortunately we can't just diff as our ppu counter is a bit different.

Reply:
Actually, will fix now since it was approved already :)